packages/app/src/files-manager/constants/files-summary-elements.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import { Type } from "Renderer/components/core/icon/icon.config"
+
+interface Item {
+  filesType: string
```

Comment:
🟡
Please refactor `filesType` to enum. It will be easier to maintenance.

Reply:
Enum filesType added in [42dd729](https://github.com/mudita/mudita-center/pull/598/commits/42dd7294f4da56efac333877d32fd5442ff68173)